api/src/main/java/org/openmrs/api/ObsService.java

```diff
@@ -73,7 +73,18 @@
 	 */
 	@Authorized(PrivilegeConstants.GET_OBS)
 	public Obs getObsByUuid(String uuid) throws APIException;
```

Comment:
I know our current api methods have this throws clause but it's redundant since this is a runtime exception, so going forward you don't have to include it